docs/src/@primer/gatsby-theme-doctocat/live-code-scope.js

```diff
@@ -1,6 +1,8 @@
 import * as primerComponents from '@primer/components'
 import * as doctocatComponents from '@primer/gatsby-theme-doctocat'
-import Octicon, {Check, Zap, X, Search} from '@primer/octicons-react'
+import Octicon, * as Icons from '@primer/octicons-react'
 import State from '../../../components/State'
 
-export default {...primerComponents, ...doctocatComponents, State, Octicon, Check, Search, Zap, X}
+const {Check, Zap, X, Search} = Icons
+
+export default {...primerComponents, ...doctocatComponents, State, Octicon, Check, Search, Zap, X, Icons}
```

Comment:
It's annoying, but can you import the icons you're using individually like we do for the others? That way the docs show `icon={PrimitiveDot}` vs `icon={Icons.PrimitiveDot}` for consistency.

Reply:
Happy to do this, although I think I'd prefer to have *every* icon in the docs be `Icon.Whatever` for clarity; when I started using these live editors it took me ages to figure out why I could only use certain icons, and also didn't want to do so many icon imports. But don't feel super strongly about this :)

Reply:
My thinking was it would be best to emulate how users would be importing/using icons in their environments and I think most people import icons individually so they don't end up with the full icon set in their bundle 🤔